contrib/misc/bareos_pam_integration/README.rst

```diff
@@ -112,7 +116,10 @@ Reuse your existing PamConsole or create an additional one::
 As PHP does not yet support TLS-PSK, the setting ``TLS Enable = no`` is required.
 Therefore it is advised to run the Bareos Director and Bareos WebUI on the same host.
 
-You may want to add following section to your :file:`/etc/bareos-webui/directors.ini`::
+Configure the ``pam_console_name`` and the ``pam_console_password`` in :file:`/etc/bareos-webui/directors.ini`
+as defined in the Console Resource, see above.
+
+::
```

Comment:
The original text indicates, that this can (and should?) be an additional section in the configuration, so that connections via Console and PAM users are possible. Could you please rephrase the sentence to indicate this?

Reply:
Rephrased.